### DISCUSSION TITLE
GPX XML: osmand:heading

### DISCUSSION THREAD

**Discussion by marsianer** [osmandapp/OsmAnd#11350] [Q&A] [answered]
What does the XML element **osmand:heading** mean? 
Stored in the GPX file.

```
<extensions>
  <osmand:speed>1.3</osmand:speed>
  <osmand:heading>150</osmand:heading>
</extensions>
```

**Comment by vshcherb** [comment, marked as answer]
Heading is taken from device as https://developer.android.com/reference/android/location/Location#getBearing()
